styles/main.css

```diff
@@ -73,15 +85,18 @@ ul {
 body{padding-top: 0;
      padding-bottom: 0;
      }
-.mugshot{box-shadow: 2px 2px 2px 3px rgba(0, 0, 0, 0.5);
-        }
-hr {
-    width:50%;
+.mugshot img {
+    box-shadow: 2px 2px 3px rgba(0, 0, 0, 0.5);
+}
+footer hr {
+    width:50vw;
     min-width:300px;
     margin-top:3em;
     margin-bottom:2em;
-    border-top:3px solid #00ffff;
-    bottom-dotted-border:8px solid #00ffff;
+    border-top: 3px solid #00ffff;
+    border-bottom:8px dotted #00ffff;

```

Comment:
`bottom-dotted-border` is not a valid CSS property.  use   `border-bottom:8px dotted #00ffff;` instead